pre_commit/resources/hook-tmpl

```diff
@@ -40,6 +41,7 @@ fi
 {pre_push}
 
 # Run pre-commit
+args="$args --hook-stage $hook_stage"
```

Comment:
Okay, but then regarding your comment above, do you mean to change it such that `run` is called explicitly:

`pre-commit run --hook-stage commit|push`

So I would add run?